FantasyUniverse/Scenes/ResetPassword/ResetPasswordView.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ResetPasswordView.swift
+//  FantasyUniverse
+//
+//  Created by TarasPeregrinus on 25.07.2022.
+//
+
+import SwiftUI
+
+struct ResetPasswordView: View {
+    @StateObject var viewModel = ResetPasswordViewModel()
+    @Environment(\.presentationMode) var presentationMode
+    
+    var body: some View {
+        AnimatedBackground(animationName: "day-background") {
+            GeometryReader { reader in
+                let animationSideSize = reader.size.width - 20
+                ScrollView {
+                    VStack {
+                        Spacer()
+                        successAnimation
+                            .frame(width: animationSideSize,
+                                   height: animationSideSize,
+                                   alignment: .center)
+                        infoLabel
+                            .frame(width: animationSideSize,
+                                   height: UIValues.defaultInfoLabelHeight,
```

Comment:
What is the usecase here? You wouldn't constrain label's height in case it would be much longer with future text changes, in other languages or someone's using a bigger font in their system.

Reply:
Right